src/benchmark.cc

```diff
@@ -67,7 +67,11 @@ DEFINE_string(benchmark_format, "tabular",
               "The format to use for console output. Valid values are "
               "'tabular', 'json', or 'csv'.");
 
-DEFINE_bool(color_print, true, "Enables colorized logging.");
+DEFINE_string(color_print, "auto",
```

Comment:
I think the three states are necessary: it's sometimes useful to pipe to some command that *does* support colorized output. In this case, the `isatty()` test would fail, so you do need a way to forcibly enable colors.